.github/workflows/update-component-versions.yaml

```
@@ -60,7 +60,7 @@ jobs:
         shell: bash
         run: |
           sed -i "s/${OLDNUM}/${NEWNUM}/g" docs/deployment/cloud.rst
-          sed -i "s/${OLDNUM}-py310/${NEWNUM}-py310/g" docs/deployment/deploying_custom_image_function.rst
+          sed -i "s/${OLDNUM}/${NEWNUM}/g" docs/deployment/deploying_custom_image_function.rst
```

let's just do the easy way for now

```suggestion
          sed -i "s/${OLDNUM}-py310/${NEWNUM}/g" docs/deployment/deploying_custom_image_function.rst
```